pkg/kubepkg/kubepkg.go

```diff
@@ -587,6 +599,7 @@ func GetDependencies(packageDef *PackageDefinition) (map[string]string, error) {
 		deps["kubelet"] = minimumKubernetesVersion
 		deps["kubectl"] = minimumKubernetesVersion
 		deps["cri-tools"] = minimumCRIToolsVersion
+		deps["kubernetes-cni"] = MinimumCNIVersion // deb based kubeadm still requires kubernetes-cni
```

Comment:
You'll need to restore variable references in the package templates as well. They should roughly match what was removed in https://github.com/kubernetes/release/pull/1330.

Reply:
Hm, when I'm now thinking about this: the `kubelet` package already provides kubernetes-cni so we may remove it completely from the deb specs. I'll change the PR to propose this approach.

Reply:
The packages should exactly match what's defined in packages/.
Let's just fix the tests here. No spec refactors.

Reply:
Okay I reverted my previous change :+1: 



> You'll need to restore variable references in the package templates as well. They should roughly match what was removed in #1330.

Which references do you mean? I'm not sure which paths needs further adaptions. :thinking: